fe/src/main/java/org/apache/doris/catalog/Catalog.java

```diff
@@ -4109,6 +4093,13 @@ private void createTablets(String clusterName, MaterializedIndex index, ReplicaS
                     db.writeUnlock();
                 }
             }
+
+            // firstColocateTable is true, means this may be the first table of colocation group,
+            // or this is just a normal table.
+            boolean firstColocateTable = backendsPerBucketSeq == null || backendsPerBucketSeq.isEmpty();
```

Comment:
the variate name `firstColocateTable ` would better change: shouldChooseBackendsRandom?  chooseBackendsFromColocateMeta?  or  isColocateChildTable ?

Reply:
changed to 'chooseBackendsArbitrary'

Reply:
OK